recipes/ifaa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ifaa" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ifaa-{{ version }}.tar.gz
+  sha256: fc3f148a91e8cb03d8690fd78c8f70d27ff20633e1db3a13eec9bf8c6aead166
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+
+
+requirements:
+  build:
+    - python
+  host:
+    - pip
+    - python
+  run:
```

Comment:
These need the minimum versions specified upstream. I would recommend using Graykull here as that also formats some other parts of your recipe.